nodes/control_node.py

```diff
@@ -99,7 +99,13 @@ def stanley_control(self, state, cx, cy, cyaw, last_target_idx):
         return delta, current_target_idx, error_front_axle
 
     def calculate_control(self):
-        if self.trajec is not None or self.trajec:
+        if (
+            self.trajec is not None
+            and self.trajec.x
+            and self.trajec.y
+            and self.trajec.theta
+            and self.trajec.v
```

Comment:
Shouldn't this also check `c` and `s`?
```suggestion
            and self.trajec.c
            and self.trajec.v
            and self.trajec.s
```

Reply:
apparently not needed